docs/monitoring/connectivity.md

```diff
@@ -8,6 +8,6 @@ We measure the connectivity from the shoot to the API Server. This is done via t
 
 ## Seed Connectivity
 
-In addition to the shoot connectivity, we also measure the seed connectivity. This means trying to reach the API Server from the seed via its load balancer. The connectivity is also displayed in the above panel as the `seed` line. Both `seed` and `shoot` connectivity are shown below.
+In addition to the shoot connectivity, we also measure the seed connectivity. This means trying to reach the API Server from the seed via the external fully qualified domain name of the API server. The connection then goes through the single shared load balancer and is routed by the envoy proxy based on the Server Name Indication TLS handshake field to the API server. The connectivity is also displayed in the above panel as the `seed` line. Both `seed` and `shoot` connectivity are shown below.
```

Comment:
```
The connection then goes through the single shared load balancer and is routed by the envoy proxy based on the Server Name Indication TLS handshake field to the API server.
```

This is true only when the APIServerSNI feature gate is enabled. So, I would rather propose to do not add such detail here. Otherwise we have to explain what is the flow when APIServerSNI is enabled and disabled.

Reply:
I removed the SNI related sentence.